javascript/javascript1/week2/other_task.js

```diff
@@ -0,0 +1,176 @@
+
+
+// Event application
+function getEventWeekday(day){
+  const weekDays=['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const currentDate  = new Date();
+  const dayNumbers = currentDate.getDay();
+   const futureDay= (day + dayNumbers) % 7;
+   console.log(weekDays[futureDay]);
+}
+getEventWeekday(1);
+
+
+// Weather wear 
+ function SuitableClothes(temperture){
+  let suggestion=``;
+  if(temperture<0){
+    suggestion=`It is very cold today, so you should wear very warm clothes`;
+  }
+  else if (temperture>=0 && temperture<10){
+    suggestion= `It is cold today, so you should wear warm clothes`;
+  }
+  else if (temperture>=10 && temperture<15){
+    suggestion=`The weather is cool, so wear relatively warm clothes`;
+  }
+  else if ( temperture>=15 && temperture<25){
+    suggestion=`The weather is quite mild; it's neither too chilly nor too warm, so you can wear spring attire`;
+  }
+  else if (temperture>=25 && temperture<35){
+    suggestion=`Today is very hot; you should wear light and cotton clothes.`;
+  }
+  else if (temperture>=35){
+   suggestion=`It's so hot today that you shouldn't wear anything \u{1f600}`;
+  }
+  return(suggestion)
+ }
+ console.log(SuitableClothes(10));
+
+
+ // switch
+ function SuitableClothes(temperture) {
+  let suggestion = ``;
+  switch (true) {
+    case temperture < 0:
+      suggestion = `It is very cold today, so you should wear very warm clothes.`;
+      break;
+    case temperture >= 0 && temperture < 10:
+      suggestion = `It is cold today, so you should wear warm clothes.`;
+      break;
+    case temperture >= 10 && temperture < 15:
+      suggestion = `The weather is cool, so wear relatively warm clothes.`;
+      break;
+    case temperture >= 15 && temperture < 25:
+      suggestion = `The weather is quite mild; it's neither too chilly nor too warm, so you can wear spring attire.`;
+      break;
+    case temperture >= 25 && temperture < 35:
+      suggestion = `Today is very hot; you should wear light and cotton clothes.`;
+      break;
+    case temperture >= 35:
+      suggestion = `It's so hot today that you shouldn't wear anything \u{1f600}`;
+      break;
+  }
+
+  return suggestion;
+}
+console.log(SuitableClothes(10));
+
+
+//Student manager
+ 
+const class07Students = [];
+function addStudent(studentName){
+  if(studentName===`Queen`){
+    class07Students.push(`Queen`)
+    return;
+  }
+  
+  if (studentName===``){
+    console.log(`Please put valid name`);
+    return;
+  }
+  
+  
+  if(class07Students.length>=5 ){
```

Comment:
Nice structure of the function with early-exiting and such! 🤩 However, it does look like people are being let into the class even if they are told there's no room! 😮 Can you spot the bug? 🐛 (Hint: what happens when `name7` is added below?)

I believe the intended behavior of `getNumberOfStudents` is to simply get the total number of students in the class, but what you have implemented is also interesting! 😄